core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -53,12 +48,4 @@ def get_gcs_resource_bucket_name():
     Returns:
         str. The bucket name for the application's GCS resources.
     """
-    if constants.EMULATOR_MODE:
-        return get_default_gcs_bucket_name()
-    else:
-        return get_application_id() + _GCS_RESOURCE_BUCKET_NAME_SUFFIX
-
-
-def get_default_gcs_bucket_name():
-    """Gets the default Google Cloud Storage bucket name for the app."""
-    return app_identity.get_default_gcs_bucket_name()
+    return get_application_id() + _GCS_RESOURCE_BUCKET_NAME_SUFFIX
```

Comment:
How would this work on a local machine? Maybe add a comment explaining (possibly in get_application_id()).

Reply:
Done, updated the docstring in `get_application_id`.